openapi-definition.yaml

```diff
@@ -0,0 +1,284 @@
+      description: Folketrygdloven $4-4
+      url: https://lovdata.no/lov/1997-02-28-19/§4-4
+  - name: Dagpengegrunnlag
+    description: Beregnger dagpengegrunnlag etter §4-11
+    externalDocs:
+      description: Folketrygdloven $4-11
+      url: https://lovdata.no/lov/1997-02-28-19/§4-11
+paths:
+  /minsteinntekt:
+    post:
+      tags:
+        - Minsteinntekt
+      summary: Kjør en beregning av minsteinntekt
+      requestBody:
+        description: Hvilke parametere som skal brukes for beregningen
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/MinsteinntektParametere'
+        required: true
+      responses:
+        200:
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MinsteinntektBeregning'
+        400:
+          description: Invalid input
+        500:
+          description: System error
+
+  /dagpengegrunnlag:
+    post:
+      tags:
+        - Dagpengegrunnlag
+      summary: Beregn dagpengegrunnlag
+      requestBody:
+        description: Hvilke parametere som skal brukes for beregningen
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DagpengegrunnlagParametere'
+            examples:
+              grunnlag:
+                summary: Eksempel for å fastsette både grunnlag og sats uten særvilkår
+                value:
+                  aktorId: 01019955667
+                  vedtakId: 1234
+                  beregningsdato: '2019-01-11'
+
+              verneplikt:
+                summary: Eksempel for å fastsette både grunnlag og sats med særvilkår
+                value:
+                  aktorId: 01019955667
+                  vedtakId: 1234
+                  beregningsdato: '2019-01-11'
+                  harAvtjentVerneplikt: true
+
+              bruktInntekt:
+                summary: Eksempel for å fastsette både grunnlag og sats med fratrekk fra tidligere brukt inntekt
+                value:
+                  aktorId: 01019955667
+                  vedtakId: 1234
+                  beregningsdato: '2019-01-11'
+                  bruktInntektsPeriode:
+                    foersteMaaned: 2019-01
+                    sisteMaaned: 2018-01
+
+              bareSats:
+                summary: Eksempel for å fastsette kun sats
+                value:
+                  aktorId: 01019955667
+                  vedtakId: 1234
+                  beregningsdato: '2019-01-11'
+                  antallBarn: 0
+                  grunnlag: 250250
+
+      responses:
+        200:
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DagpengegrunnlagBeregning'
+
+components:
+  schemas:
+
+    Subsumsjon:
+      type: object
+      required:
+        - beregningsId
+        - opprettet
+        - utført
```

Comment:
I see you have avoided using æøå in other names, should this be changed to 'utfort'?

Reply:
Good point 👍 

```suggestion
        - utfort
```